packages/dendron-plugin-views/src/hooks/useSyncGraphWithIDE.tsx

```diff
@@ -47,10 +47,6 @@ const useSyncGraphWithIDE = ({ graph, ide, engine, config }: Props) => {
       ) {
         graphActiveNode.select();
         graph.center(graphActiveNode);
-        graph.zoom({
```

Comment:
This is how the graph looks like for me when I open it:

![2022-05-09-17-53-48-983259841](https://user-images.githubusercontent.com/1008124/167505131-02eb35af-d190-4f2b-84c8-b1525ffa9447.png)

Would it make sense to add back the initial zoom here? I think it might make more sense if it was at least zoomed in enough to display the current note.

Reply:
The only reason behind removing it was, that it gets way too zoomed in. It gives an impression that the graph isn't loaded but when we zoom out we see all the nodes. Will find an optimum zoom value for this an update.